app/View/Components/Platform/Cards/Game.php

```diff
@@ -0,0 +1,304 @@
+    private array $userGameProgressionAwards;
+    private ?string $usernameContext;
+
+    public function __construct(int $gameId, ?string $targetUsername = null)
+    {
+        $this->gameId = $gameId;
+        $this->usernameContext = $targetUsername ?? Auth::user()->User ?? null;
+    }
+
+    public function render(): ?View
+    {
+        $rawGameData = $this->getGameData($this->gameId);
+
+        if (!$rawGameData) {
+            return null;
+        }
+
+        $this->userGameProgressionAwards = $this->getUserGameProgressionAwards(
+            $this->gameId,
+            $this->usernameContext
+        );
+
+        $cardViewValues = $this->buildAllCardViewValues(
+            $rawGameData,
+            $this->userGameProgressionAwards
+        );
+
+        return view('platform.components.cards.game', $cardViewValues);
+    }
+
+    /**
+     * Retrieves the game data for a given game ID.
+     * It first checks if the data is available in cache. If it is, the cached data is returned.
+     * Otherwise, the method fetches the data from the database, caches it, and then returns it.
+     *
+     * The fetched game data includes game metadata, console info, core achievements, dev claims,
+     * and alternate games (used if the game is for a hub).
+     *
+     * @return ?array<mixed> The game data. Returns null if no game is found with the given game ID.
+     */
+    private function getGameData(int $gameId): ?array
+    {
+        $cacheKey = CacheKey::buildGameCardDataCacheKey($gameId);
+
+        if (Cache::store('array')->has($cacheKey)) {
+            return Cache::store('array')->get($cacheKey);
+        }
+
+        $loadGameCardData = (function () use ($gameId): ?array {
+            $foundGame = GameModel::with([
+                'system',
+                'achievements' => function ($query) {
+                    $query->published();
+                },
+            ])->find($gameId);
+
+            if (!$foundGame) {
+                return null;
+            }
+
+            $foundGameConsoleId = $foundGame->system->ID;
+            $foundGameAchievements = $foundGame->achievements->toArray();
+
+            $foundClaims = AchievementSetClaim::where('GameID', $gameId)->get()->toArray();
+
+            $foundAltGames = [];
+            if ($foundGameConsoleId === $this->hubConsoleId) {
+                $foundAltGames = GameAlternative::where('gameID', $gameId)->get()->toArray();
+            }
+
+            return array_merge(
+                $foundGame->toArray(), [
+                    'ConsoleID' => $foundGameConsoleId,
+                    'ConsoleName' => $foundGame->system->Name,
+                    'Achievements' => $foundGameAchievements,
+                    'Claims' => $foundClaims,
+                    'AltGames' => $foundAltGames,
+                ]
+            );
+        })();
+
+        Cache::store('array')->put($cacheKey, $loadGameCardData, Carbon::now()->addDays(7));
```

Comment:
Adding a TTL using the ephemeral `array` cache won't have an effect.

Reply:
Understood, I've removed it in https://github.com/RetroAchievements/RAWeb/pull/1700/commits/a2da137cb9554818c12868ad94e986db44a26a49.

Should it also be removed from _UserCard.php_ here?: https://github.com/RetroAchievements/RAWeb/blob/515265d2cdf80d325f0d2a98521f775e9805143c/app/View/Components/Community/UserCard.php#L60

Reply:
that'd work as it won't have an effect there either 👍

Reply:
No problem at all, addressed in https://github.com/RetroAchievements/RAWeb/pull/1700/commits/59f3825106064a9ce969faace1c0b20ed7449d6c.